modules/claims_api/lib/bgs_service/local_bgs.rb

```diff
@@ -14,9 +14,30 @@ module ClaimsApi
   class LocalBGS
     attr_accessor :external_uid, :external_key
 
+    # rubocop:disable Metrics/MethodLength
     def initialize(external_uid:, external_key:)
+      @client_ip =
+        if Rails.env.test?
+          # For all intents and purposes, BGS behaves identically no matter what
+          # IP we provide it. So in a test environment, let's just give it a
+          # fake so that cassette matching isn't defeated on CI and everyone's
+          # computer.
+          '127.0.0.1'
```

Comment:
this is needed for the body matching (headers built in to the request) to align with the new `xml_body` matching in cassettes?

Reply:
That's right, this value goes in the body of the request to BGS, but it is a value that varies between CI and localhost, so a cassette developed locally won't work correctly when running in CI. Making the value static in test allows CI and localhost to match on XML body.